.github/workflows/ci-testing-deploy.yml

```
@@ -2864,3 +2864,56 @@ jobs:
         env:
           TAG_PREFIX: hotfix-staging-github
         run: ./ci/deploy/dockerhub-deploy.bash -n
+
+  check-api-specs-updated:
+    needs: [changes]
+    if: ${{ needs.changes.outputs.anything-py == 'true' || github.event_name == 'push' }}
+    timeout-minutes: 10
+    name: check api-specs are up to date
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup python environment
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: install uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: "0.4.x"
+          enable-cache: false
+      - name: checkout source branch
+        uses: actions/checkout@v4
+      - name: Regenerate specs and check
+        run: |
+          uv venv .venv && source .venv/bin/activate
+          make openapi-specs
+          changed_files=$(git diff --name-only | awk '{$1=$1};1')
+          if [[ -n "${changed_files}" ]]; then
+            echo -e "Run 'make openapi-specs' to update the following specs: \n${changed_files}"
+            exit 1
+          fi
+
+
+  api-spec-backwards-compatible:
+    needs: [changes]
+    if: ${{ needs.changes.outputs.anything-py == 'true' || github.event_name == 'push' }}
+    timeout-minutes: 10
+    name: "api-specs-backwards-compatibility"
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup python environment
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: install uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: "0.4.x"
+          enable-cache: false
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: Check openapi-specs backwards compatibility
+        run: |
+          bash ci/github/helpers/openapi-specs-backwards-compatibility.bash \
```

I think should work as well
```suggestion
          ./ci/github/helpers/openapi-specs-backwards-compatibility.bash \
```

---

fixed